jwql/website/apps/jwql/static/js/jwql.js

```diff
@@ -469,23 +469,27 @@ function sort_by_proposals(sort_type) {
 
 
 /**
- * Sort thumbnail display by a given sort type
- * @param {String} sort_type - The sort type (e.g. file_root", "exp_start")
+ * Sort thumbnail display by a given sort type, save sort type in session for use in previous/next buttons
+ * @param {String} sort_type - The sort type by file name
  */
-function sort_by_thumbnails(sort_type) {
+function sort_by_thumbnails(sort_type, base_url) {
 
     // Update dropdown menu text
     document.getElementById('sort_dropdownMenuButton').innerHTML = sort_type;
 
     // Sort the thumbnails accordingly
     var thumbs = $('div#thumbnail-array>div')
-    if (sort_type == 'Name') {
-        tinysort(thumbs, {attr:'file_root'});
-    } else if (sort_type == 'Default') {
-        tinysort(thumbs, {selector: 'img', attr:'id'});
-    } else if (sort_type == 'Exposure Start Time') {
-        tinysort(thumbs, {attr:'exp_start'});
+    if (sort_type == 'Ascending') {
+        tinysort(thumbs, {attr:'file_root', order:'asc'});
+    } else if (sort_type == 'Descending') {
+        tinysort(thumbs, {attr:'file_root', order:'desc'});
     }
+    $.ajax({
```

Comment:
@BradleySappington this is clever. Have you seen what happens where the is a failure? Is anything reported to the page for the user to see?

Reply:
It will report it only in the console, so if you have the console up on your terminal you will see it.  I didn't think a session not-updating is something that the user should be aware of.